### DISCUSSION TITLE
Import error

### DISCUSSION THREAD

**Discussion by ak061985** [liiight/notifiers#463] [Q&A]
Notifiers no imported get_notifier. "Cannot find reference 'get_notifier' in '__init__.py'". If im use "note = notifiers.get_notifier" PyCharm say me "AttributeError: partially initialized module 'notifiers' has no attribute 'get_notifier' (most likely due to a circular import)"

**Comment by ak061985** [comment]
And 
File "E:\......\venv\lib\site-packages\referencing\_core.py", line 55, in Specification
    ] = field(alias="anchors_in")
TypeError: field() got an unexpected keyword argument 'alias'

**Comment by liiight** [comment]
seems like a bad install. how did you install? which python version?

**Reply by ak061985** [reply]
pip install notifiers, Python 3.10.8

**Comment by liiight** [comment]
please share the entirety of the error(s) you get

**Comment by ak061985** [comment]
If im install in my telegram bot 

![2023-09-06_07-34-52](https://github.com/liiight/notifiers/assets/144016162/e08ad81b-86b0-4c74-8e00-3ef5ebc4cbad)

If I install in a separate project, then everything is fine

**Reply by ShoGinn** [reply]
It looks like you have `notifiers.py` file in that same directory @ak061985; thats why you are getting the error.
